docker/geant4/Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN mkdir -p /opt/geant4/src && \
     cmake -DCMAKE_INSTALL_PREFIX=/opt/geant4/install \
           -DGEANT4_BUILD_MULTITHREADED=ON \
           -DGEANT4_INSTALL_EXAMPLES=OFF \
+          -DGEANT4_USE_QT=ON \
```

Comment:
According to the G4 instructions, one should also add the `-DCMAKE_PREFIX_PATH=/path/to/Qt/installation` flag. Does it work without it?

Reply:
Since Qt was in a standard path, cmake was able to find it automatically. In any, case, I have check it running the docker container and, as you can see, it has Qt enabled and the proper paths were included:

```
root@6554bee89f16:/opt/geant4/install/bin# geant4-config  --cflags
-DG4VIS_USE_OPENGL -DG4UI_USE_TCSH -DG4UI_USE_QT -DG4VIS_USE_OPENGLQT -I/usr/include/x86_64-linux-gnu/qt5/ -I/usr/include/x86_64-linux-gnu/qt5/QtCore -I/usr/lib/x86_64-linux-gnu/qt5//mkspecs/linux-g++ -I/usr/include/x86_64-linux-gnu/qt5/QtGui -I/usr/include/x86_64-linux-gnu/qt5/QtWidgets -I/usr/include/x86_64-linux-gnu/qt5/QtOpenGL -I/usr/include/x86_64-linux-gnu/qt5/QtPrintSupport -W -Wall -pedantic -Wno-non-virtual-dtor -Wno-long-long -Wwrite-strings -Wpointer-arith -Woverloaded-virtual -Wno-variadic-macros -Wshadow -pipe -pthread -ftls-model=initial-exec -std=c++17 -I/opt/geant4/install/bin/../include/Geant4
```